xml/article_sap_trento.xml

```diff
@@ -356,6 +356,13 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
           linkend="sec-trento-server-requirements" /> for minimum requirements)
         and follow steps in <xref linkend="pro-trento-manually-installing" /> to
         get &t.server; up and running. </para>
+     
+     <important>
+      <title>Deploying Trento on K3s</title>
+      <para>
+        Please note that SUSE does not recommend using Trento on K3s for production purposes.
```

Comment:
I have only these minor issues:

* We don't use "please" in our documentation.
* As this is an important note, the "note" here seems unnecessary to me.
* SUSE -> `&suse;`

I'd suggest to use:

```suggestion
        &suse; does not recommend using Trento on K3s for production purposes.
```